Resources/Prototypes/Roles/Jobs/Security/security_officer.yml

```diff
@@ -29,3 +29,4 @@
   storage:
     back:
     - Flash
+    - MagazinePistolSP
```

Comment:
please make these part of the loadout itself, so different guns define different magazines